lib/produce/itunes_connect.rb

```diff
@@ -92,19 +107,30 @@ def initial_create
     end
 
     def initial_pricing
+      # It's not important here which is the price set. It will be updated by deliver
       sleep 3
       click_on "Pricing"
-      first('#pricingPopup > option[value="3"]').select_option
+      first('#pricingPopup > option[value="0"]').select_option
       first('.saveChangesActionButton').click
     end
 
     private
+      def bundle_exist?
+        if @wildcard_bundle
+          return false # We should always create the app from scratch.
```

Comment:
I'm sorry, could you comment (here, in discussion) please why we should always create the app, shouldn't we just behave as if `@full_bundle_identifier == @config[:bundle_identifier]`? Or, am I missing something?

I'm sorry if it's that new bundled apps thing. Didn't have a chance to work with them.